### PR TITLE
Add setup-cxx action to install gcc-8 on bionic

### PR DESCRIPTION
**Pull request by auscompgeek**
Test run on https://github.com/robotpy/robotpy-wpiutil/pull/28

### CODE REVIEW

File: setup-cxx/setup.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+if [ "$RUNNER_OS" != "Linux" ]; then
+	echo "On $RUNNER_OS, doing nothing."
+	exit
+fi
+
+sudo apt-get install --no-install-recommends g++-8 gcc-8
+
+DIR="$RUNNER_TEMP/gcc-override"
+mkdir "$DIR"
+ln -s /usr/bin/gcc-8 "$DIR/gcc"
```
Comment (virtuald) [inline setup-cxx/setup.sh:14]: Isn't there a command for setting this up?

Comment (auscompgeek) [reply setup-cxx/setup.sh:14]: It doesn't look like `/usr/bin/gcc` ordinarily participates in Debian's alternatives system.

**Comment by auscompgeek** [comment]
Shall we just do this for now, or do we want to go down the container image route?

**Comment by virtuald** [comment]
Just do this for now if it works. It would be best if this were moved to a reusable workflow...

**Pull request merged**